src/main/java/core/basesyntax/service/HandlersStorage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.operation.OperationBHandler;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.service.operation.OperationPHandler;
+import core.basesyntax.service.operation.OperationRHandler;
+import core.basesyntax.service.operation.OperationSHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HandlersStorage {
+    private final Map<String, OperationHandler> handlers = new HashMap<>();
+
+    {
+        handlers.put("b",new OperationBHandler());
```

Comment:
в тебе тут доречі в деяких хендлерах однаковий код, що уникнути дублювання ти можеш підкорегувати неймінг і використати один хендлер для двох і більше операцій
```suggestion
        handlers.put("r",new IncreaseFruitHandler());
        handlers.put("s",new IncreaseFruitHandler());
```